### PR TITLE
chore(cloud-bench): adds failure tolerance and max concurrent count

### PR DESCRIPTION
**Pull request by sameer-in**

### CODE REVIEW

File: modules/services/cloud-bench/main.tf [hunk 1]
```diff
@@ -162,4 +162,10 @@ resource "aws_cloudformation_stack_set_instance" "stackset_instance" {
   deployment_targets {
     organizational_unit_ids = [for root in data.aws_organizations_organization.org[0].roots : root.id]
   }
+  operation_preferences {
+    failure_tolerance_count = 100
+    # failure_tolerance_percentage = 100
```
Comment (nkraemer-sysdig) [inline modules/services/cloud-bench/main.tf:167]: Typo?

Comment (sameer-in) [reply modules/services/cloud-bench/main.tf:167]: Wanted to mention there is another option as percentage. should we remove those? @nkraemer-sysdig

Comment (nkraemer-sysdig) [reply modules/services/cloud-bench/main.tf:167]: Oh I see. Yes, maybe lets remove it if we're not using percentages

Comment (sameer-in) [reply modules/services/cloud-bench/main.tf]: done. removed them.

**Review by nkraemer-sysdig — APPROVED** [review]

**Review by haresh-suresh — APPROVED** [review]

**Comment by wideawakening** [comment]
if  `CI - Pull Request / Max TF pre-commit` gets stuck its sometimes because there has been a version bump and we have to change it in the gh repo settings. gotta find a way to avoid it.

bumped and now its good to merge

**Pull request merged**